pkg/kubelet/kubelet.go

```diff
@@ -1610,11 +1610,7 @@ func (kl *Kubelet) syncPod(o syncPodOptions) error {
 	}
 
 	// Wait for volumes to attach/mount
-	defaultedPod, _, err := kl.defaultPodLimitsForDownwardApi(pod, nil)
```

Comment:
This method is still being used elsewhere right? Is the actual issue with this method?